### ISSUE TITLE
Improve Cypress GitHub Action

### ISSUE THREAD

**Issue by Phixyn** [Phixyn/react-todo-app#9]
Use the Cypress GitHub Action to test on multiple browsers. We could also try to run tests in parallel. Check out [docs here](https://docs.cypress.io/guides/continuous-integration/github-actions.html#Cypress-GitHub-Action).

**Comment by Phixyn** [comment]
Parallelization requires Cypress Cloud. Running Cypress on multiple browsers seems overkill for this project. Won't do.
